portfolio/src/main/webapp/aboutMe.html

```diff
@@ -2,21 +2,31 @@
 <html>
   <head>
     <meta charset="UTF-8">
+
+    <!--Borders for paragraph and header -->
+    <style>
+    p.solid {border-style: solid;}
+    h1.solid {border-style: solid;}
+    </style>
```

Comment:
Would be best to add this to the style.css files. That way all styles are kept in one place.